en/installation-guide/system/database-initialization.rst

```diff
@@ -13,9 +13,29 @@ In pgAdminIII
 
 * Connect to the database server
 
-* Create a new login role (preferred name `qgep`, but you are free to choose)
+* Create a new group role (preferably named `qgep`, but you are free to choose)
 
-* Create a new database (preferred name `qgep`, but you are free to choose)
+* Create a new login role (`qgepuser` for example) and make it a member of the `qgep` group
+
+You can do this graphically or by simply opening a pgAdminIII SQL Window and type the following:
+
+::
+
+ CREATE GROUP qgep;
+ CREATE ROLE qgepuser LOGIN;
+ GRANT qgep TO qgepuser;
+
+.. figure:: images/001_roles.jpeg
```

Comment:
Nice. Do you think it would be good to add a separate chapter concerning PgAdminIII and link to this page wherever something requires running an SQL query?